### PR TITLE
Add Tab Switcher button on the bottom toolbar

### PR DESCRIPTION
**Pull request by codeimpl**
Tab Switcher button exists on the top toolbar. But it is not easy to
handle it. So this patch adds the button to the bottom toolbar.

Refs: #772
Co-authored-by: Jaeyong Bae <jdragon.bae@gmail.com>

### CODE REVIEW

File: src/org/triple/banana/toolbar/BottomToolbarController.java [hunk 1]
```diff
@@ -99,9 +101,7 @@ private ToolbarButton createToolbarButton(ButtonId buttonId) {
                 new ViewGroup.LayoutParams(width, ViewGroup.LayoutParams.MATCH_PARENT));
         toolbarButton.setButtonId(buttonId);
         toolbarButton.setImageResource(ButtonId.getImageResource(buttonId));
-        if (ButtonId.getOnClickListeners(buttonId) != null) {
-            toolbarButton.setOnClickListener(ButtonId.getOnClickListeners(buttonId));
-        }
+
```
Comment (romandev) [inline src/org/triple/banana/toolbar/BottomToolbarController.java:104]: Unnecessary blank

File: src/org/triple/banana/toolbar/TabSwitcherToolbarButton.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    @Override
+    public void onTintChanged(ColorStateList tint, boolean useLight) {
+        super.onTintChanged(tint, useLight);
+
+        if (mTabSwitcherButtonDrawable != null) {
+            mTabSwitcherButtonDrawable.setTint(tint);
+        }
+    }
+}
+
```
Comment (romandev) [inline src/org/triple/banana/toolbar/TabSwitcherToolbarButton.java:68]: ditto

**Review by romandev — COMMENTED** [review]
Please run `git cl format`

**Comment by romandev** [comment]
@codeimpl, @jdragonbae

File: src/org/triple/banana/toolbar/BottomToolbarController.java (hunk 1 above)
Comment (jdragonbae) [reply src/org/triple/banana/toolbar/BottomToolbarController.java]: Thanks. Done.

File: src/org/triple/banana/toolbar/TabSwitcherToolbarButton.java (hunk 2 above)
Comment (jdragonbae) [reply src/org/triple/banana/toolbar/TabSwitcherToolbarButton.java]: Done.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**